.github/renovate.json5

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":dependencyDashboard",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    ":disableMajorUpdates",
```

Comment:
I recommend against this setting. You'll lose all visibility of major updates this way as well as the ability for Renovate to ever automate PRs when you're ready. You're better to simply keep them as requiring approval so that you can be aware of what's required updating and trigger PRs on-demand.

Reply:
I agree with this.

Reply:
I see, makes sense to me.